File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    let answer = 0;
    const bioindex = binomial.split(" ");
    // binomial을 배열로 변환한다
    
    const a = parseInt(bioindex[0]);
    const op = bioindex[1];
    const b = parseInt(bioindex[2]);
    // parseInt를 사용하여 문자열을 정수로 변환한다
    
    if (op === "+") {
        answer += a + b;
    } // op가 +라면 a + b를 수행하고  
    else if (op === "-") {
        answer += a - b;
    } // op가 -라면 a - b를 수행하고 
    else if (op === "*") {
        answer += a * b;
    } // op가 *라면 a * b를 수행한다

    return answer;
}

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        int answer = 0;
        String[] bioindex = binomial.split(" ");
        // binomial을 배열로 변환한다
        
        int a = Integer.parseInt(bioindex[0]);
        String op = bioindex[1];
        int b = Integer.parseInt(bioindex[2]);
        // 숫자는 각 a,b 기호는 op로 정한다

        if (op.equals("+")) {
            answer += a + b;
        } // op가 +라면 a + b를 수행하고 
        else if (op.equals("-")) {
            answer += a - b;
        } // op가 -라면 a - b를 수행하고
        else if (op.equals("*")) {
            answer += a * b;
        } // op가 *라면 a * b를 수행한다
        return answer;
    }
}